DEPS

```diff
@@ -496,7 +496,7 @@ deps = {
    Var('dart_git') + '/external/github.com/google/webkit_inspection_protocol.dart.git' + '@' + Var('dart_webkit_inspection_protocol_rev'),
 
   'src/third_party/dart/third_party/pkg/yaml':
-   Var('dart_git') + '/yaml.git@9f0d64934c07bc27438074616455618b7103582d',
+  'https://github.com/yaml/pyyaml.git@155ec463f6a854ac14ccd5e2dda8017ce42a508a',
```

Comment:
A bit confused by this change since it is replacing the Dart yaml library with the Python yaml library. Was this supposed to replace some other dependency?

Also, my understanding is that we should still be moving away from pulling dependencies directly from github.